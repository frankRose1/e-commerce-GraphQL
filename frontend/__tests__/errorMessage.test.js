import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import ErrorMessage from '../components/ErrorMessage';

const mockSingleError = {
  message: 'GraphQL error: This is a test error'
};

describe('ErrorMessage />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<ErrorMessage error={mockSingleError} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
