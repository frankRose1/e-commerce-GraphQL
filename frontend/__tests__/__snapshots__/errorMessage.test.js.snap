// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage /> renders and matches the snapshot 1`] = `
<ErrorMessage__ErrorStyles>
  <p
    data-test="graphql-error"
  >
    <strong>
      Shoot!
    </strong>
    This is a test error
  </p>
</ErrorMessage__ErrorStyles>
`;
